concepts/05 Widgets/DataGrid/20 Editing/20 API/40 Get Current Cell Values.md

```diff
@@ -70,6 +70,95 @@ The **cellValue(rowIndex, dataField)** method requires a row index. Use the [get
     })
     export class AppModule { }
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDataGrid ...
+            :ref="dataGridRefKey"
+            @editing-start="onEditingStart">
+        </DxDataGrid>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxDataGrid from 'devextreme-vue/data-grid';
+
+    const dataGridRefKey = 'myDataGrid';
+
+    export default {
+        components: {
+            DxDataGrid
+        },
+        data: function() {
+            return {
+                dataGridRefKey,
+                currentRowIndex
+            }
+        },
+        methods: {
+            onEditingStart(e) {
+                this.currentRowIndex = e.component.getRowIndexByKey(e.key);
```

Comment:
It will be better to save key in data because row index can be changed during editing. For example, if virtual scrolling is used or if some row before editing row is expanded during editing. 
```
onEditingStart(e) {
    this.editRowKey = e.key;
    // ...
},
getCellValue() {
    const editRowIndex = e.component.getRowIndexByKey(this.editRowKey);
    if(editRowIndex >= 0) {
        this.dataGrid.cellValue(editRowIndex, "EmployeeName");
    }
}
```